cluster/aws/images/k8s-ebs-jessie-amd64-hvm.yml

```diff
@@ -0,0 +1,52 @@
+---
+name: k8s-1.2-debian-{system.release}-{system.architecture}-{provider.virtualization}-{%Y}-{%m}-{%d}-ebs1
+provider:
+  name: ec2
+  virtualization: hvm
+  enhanced_networking: simple
+  description: Kubernetes 1.2 Base Image - Debian {system.release} {system.architecture}
+bootstrapper:
+  workspace: /target
+  tarball: true
+system:
+  release: jessie
+  architecture: amd64
+  #  bootloader: extlinux See https://github.com/andsens/bootstrap-vz/issues/182
+  bootloader: grub
+  charmap: UTF-8
+  locale: en_US
+  timezone: UTC
+volume:
+  backing: ebs
+  partitions:
+    type: msdos
+    root:
+      filesystem: ext4
+      size: 8GiB
+packages:
+  mirror: http://cloudfront.debian.net/debian
+  install:
+    - aufs-tools
+    - curl
+    - python-yaml
+    - git
+    - nfs-common
+    - bridge-utils
+    - logrotate
+    - socat
+    - python-apt
+    - apt-transport-https
+    - unattended-upgrades
+    - lvm2
+    - btrfs-tools
+plugins:
+  cloud_init:
+    metadata_sources: Ec2
+    username: admin
+  ntp: {}
```

Comment:
I think these only apply to GCE, not AWS?

Happy to create a GCE image as well - that's half the point - but I'd rather start with the AWS image first...

Although this does suggest it should live in `cluster/images`, not `cluster/aws`, so the files can be side-by-side